dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1346,3 +1346,14 @@ def test_moment():
     ddf = dd.from_pandas(df, npartitions=2)
 
     assert_eq(stats.moment(ddf, 2, 0), scipy.stats.moment(df, 2, 0))
+
+
+@pytest.mark.parametrize("func", ["sum", "count", "mean", "var", "sem"])
+def test_empty_df_reductions(func):
+    pdf = pd.DataFrame()
```

Comment:
Can you also test with a dataframe that as rows, but no columns?

Reply:
are you referring to something like this?
```
>>> idx = pd.date_range('2000', periods=4)
>>> pd.DataFrame(index=idx)
Empty DataFrame
Columns: []
Index: [2000-01-01 00:00:00, 2000-01-02 00:00:00, 2000-01-03 00:00:00, 2000-01-04 00:00:00]
>>> pd.DataFrame(index=idx).count()
Series([], dtype: int64)
```

Reply:
Yep.